### DISCUSSION TITLE
wLUNA Tokens never arrived on SOLANA (through wormhole)

### DISCUSSION THREAD

**Discussion by BitcoinPapi** [wormhole-foundation/wormhole#227] [Problems / bugs] [answered]
Hi all,

I  transferred some Luna tokens from ETH (metamask) to SOL (phantom) using wormhole and the $luna left my ETH address but never arrived to my SOL address.

Here is the Hash: 0x9ee767fa49d003c3b9644c5a1e9fbdeb796e41f317315c2c676c2cd666a14aa4

Here is my SOL Address: 2DeMvFQV2vHbdtCcw78sgJ8ccJ2VC1Cyav4DM7t5ucvr

Please let me know how to recover my $luna tokens. 

Thanks,

**Comment by hendrikhofstadt** [comment, marked as answer]
Completed here https://explorer.solana.com/tx/5i44jbkbSBxATqZ5RU4tXAznc6td6f6LH2QFV96dtg8mbjKpbZetYGhJPwyCZKPSJh8bCvVJuSHEiJY5nPCYdDyw

**Reply by BitcoinPapi** [reply]
Thank you so much!
